organisations/README.md

```diff
@@ -0,0 +1,122 @@
+# Organisation Mapping
+
+## XML HL7 > JSON FHIR
+
+Organisations are generated from the HL7v3 Agent Directory entries which represent either an `AgentOrganisation`
+or an `AgentPerson` with an associated `representedOrganisation`.
+
+### AgentOrganisation
+| Mapped to (JSON FHIR Organisation resource field) | Mapped from (XML HL7 / other)                                       |
+|---------------------------------------------------|---------------------------------------------------------------------|
+| id                                                | `Agent / id [@root]`                                                |
+| identifier\[0].system                             | fixed value = `"https://fhir.nhs.uk/Id/ods-organization-code"`      |   
+| identifier\[0].value                              | `Agent / agentOrganisation / id [@root]`                            |
+| name                                              | `Agent / agentOrganisation / name`                                  | 
+| type\[0].text                                     | `Agent / code / originalText` or `Agent / code [@displayName]`      |
+| address\[0].use                                   | fixed value = `"work"`                                              |
+| address\[0].type                                  | fixed value = `"physical"`                                          |
+| address\[0].line[]                                | `Agent / agentOrganisation / addr / streetAddressLine` <sup>1</sup> |
+| address\[0].postalCode                            | `Agent / agentOrganisation / addr / postalCode`                     |
```

Comment:
Musing: We use this address mapping within `Location` too, I wonder if we should have this duplication within the documentation or refactor it out.

Reply:
I'm open to either. Although the util method is only used by those to mappers, so we would have to write it again.